services/editor/src/pages/keys/components/KeysList/KeysList.js

```diff
@@ -36,8 +37,17 @@ const KeyItem = connect((state, props) => ({
   </div>,
 );
 
+const withoutInternal = list => (list ? list.filter(x => !/^@tweek\//.test(x)) : list);
+
 const KeysList = componentFromStream((prop$) => {
-  const keyList$ = prop$.map(x => x.keys).distinctUntilChanged();
+  const showInternal$ = Observable.fromPromise(
```

Comment:
maybe use:
Observable.defer(()=>fetch('/api/editor-configuration/show_internal_keys').then(response => response.json());
Instead of Observable fromPromise.

I think Observables should default to cold lazy behavior.